locales/en/rle-director.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
After the last name of the RO,  we need to add " (Born " and also a closinh bracket after the DOB.
)